androidapp/HopSquadApp/app/src/main/res/values/strings.xml

```diff
@@ -29,4 +29,17 @@
     <string name="payment_cvv">CVV</string>
     <string name="payment_expiration">Exp</string>
 
+    <string-array name="beer_order_amount">
```

Comment:
Found out there may be a TABC restriction about how many beers a single user can order, but this is a business and not technical edit. I think we can leave for now just to show.

Reply:
That is god to know.